src/dbnode/storage/limits/permits/fixed_permits_test.go

```diff
@@ -37,13 +37,16 @@ func TestFixedPermits(t *testing.T) {
 	require.NoError(t, err)
 	expectedP := NewPermit(1, iOpts)
 	expectedP.(*permit).refCount.Inc()
-	p, err := fp.Acquire(ctx)
+	r, err := fp.Acquire(ctx)
+	p := r.Permit
```

Comment:
nit: check error first in case this is nil, same for the other calls in this file

Reply:
Done.